usr/lib/systemd/system/greenboot-healthcheck.service

```diff
@@ -10,6 +10,8 @@
 [Unit]
 Description=greenboot Health Checks Runner
 Before=boot-complete.target
+After=network-online.target
+Wants=network-online.target
```

Comment:
Let's split out the dns test into a sub-package and put this change here in a systemd unit dropin, so it only get's added when the DNS test is installed. Otherwise this'll break disconnected systems.